content/posts/bitcoin.md

```diff
@@ -304,7 +306,7 @@ in the end for the better.
 
 Many people have heard the phrase "blockchains, not Bitcoin", 
 meant to express that blockchain technology is a meaningful innovation,
-but Bitcoin can be largely ignored. Of course we don't believe Bitcoin can or
+but Bitcoin can be largely ignored. Of course, I don't believe Bitcoin can or
```

Comment:
Just cause I didn't generally see third person used in this case of case early in the piece.